ext/handlers/filters/reaction/reaction.go

```diff
@@ -0,0 +1,60 @@
+package reaction
+
+import (
+	"github.com/PaulSonOfLars/gotgbot/v2"
+	"github.com/PaulSonOfLars/gotgbot/v2/ext/handlers/filters"
+)
+
+func All(_ *gotgbot.MessageReactionUpdated) bool {
+	return true
+}
+
+func FromUserID(id int64) filters.Reaction {
+	return func(mru *gotgbot.MessageReactionUpdated) bool {
+		if mru.User != nil {
+			return mru.User.Id == id
+		}
+
+		return false
+	}
+}
+
+func FromAnonymousChatID(id int64) filters.Reaction {
```

Comment:
Random thought: should we be merging this to a single item? How are the other handlers doing it? I'm not sure it makes sense to have both, since only one can trigger.

Reply:
Hm, yep, correct.

Reply:
How do you think about merge them and name as `FromPeer`? 
I don't know how Telegram names such things exactly in BotAPI, but in MTProto it's named usually as `Peer`.